core/che-core-metrics-core/src/test/java/org/eclipse/che/core/metrics/ApiResponseCounterTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.core.metrics;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.search.RequiredSearch;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class ApiResponseCounterTest {
+  ApiResponseCounter apiResponseCounter;
+  MeterRegistry registry;
+
+  @BeforeMethod
+  public void setup() {
+    registry = new SimpleMeterRegistry();
+
+    apiResponseCounter = new ApiResponseCounter();
+    apiResponseCounter.bindTo(registry);
+  }
+
+  @Test
+  public void testServerError() {
+    apiResponseCounter.handleStatus(500);
+
+    RequiredSearch meter = registry.get("che.server.api.response.success");
+    registry.getMeters();
+  }
+
+  @DataProvider(name = "information")
+  public Object[][] information() {
```

Comment:
You are going to use these data providers somewhere in tests?

Reply:
yes, tests are now finished